src/daemon.rs

```diff
@@ -78,14 +93,21 @@ pub fn notify(unset_env: bool, state: &[NotifyState]) -> Result<bool, SdError> {
     };
     let sock =
         UnixDatagram::unbound().map_err(|e| format!("failed to open datagram socket: {}", e))?;
+    sock.connect(path)
+        .map_err(|e| format!("failed to connect to notify socket: {}", e))?;
```

Comment:
send_fds_to takes a &UnixSocketAddr rather than a PathBuf so it is easier to use connect and send, send_fds instead of send_to, send_fds_to without connect.

Since the socket is never used to send messages to anywhere else the result is equivalent in this case.